cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -1089,3 +1091,58 @@ def pre_calibration(
         pre_calibration_config=pre_calibration_config,
     )
     update_table(df_for_calibration, responses_pre_calibration_table, mode_overide="overwrite")
+
+
+@register_pipeline_stage("aggregated_output")
+def aggregated_output(
+    apply_window,
+    apply_groupby,
+    aggregated_output,
+    column_name_to_assign_list,
+    column_group,
+    column_window_list,
+    function_list,
+    column_list_to_apply_function,
+):
+    """
+    Parameters
+    ----------
+    apply_window
+    apply_groupby
+    aggregated_output
+    column_name_to_assign_list
+    column_group
+    column_window_list
+    function_list
```

Comment:
Where the user passes a function, this will be a string in the config. This string should only include the function name (e.g. `max`). Your code will then need to combine this with the target column(s) to get the `F.min("col_name")`
You may want to use the `getattr()` function to fetch the function from pyspark's `F`. E.g. `getattr(F, function_name)("col_name"))`

If you do this, you should add a check to ensure that the function exists before using getattr. Raise an error if the function doesn't exist